Resources/config/services.yml

```diff
@@ -0,0 +1,13 @@
+parameters:
+    ez_privacy_cookie_twig_extension: EzSystems\PrivacyCookieBundle\Twig\PrivacyCookieTwigExtension
+
+services:
+    ez_privacy_cookie.twig.extension:
+        class: %ez_privacy_cookie_twig_extension%
+        arguments:
+            - @service_container
+        calls:
+            - [setCookieName, [%ez_privacy_cookie.cookie_name%]]
+            - [setDays, [%ez_privacy_cookie.days%]]
```

Comment:
It will be changed in the next commit. Defaults will be set by a constructor parameters.